shopify_python/google_styleguide.py

```diff
@@ -251,3 +257,31 @@ def __use_cond_expr(self, node):  # type: (astroid.If) -> None
                 else_body_name = node.orelse[0].targets[0].name
                 if if_body_name == else_body_name:
                     self.add_message('cond-expr', node=node)
+
+    def __lambda_func(self, node): # type: (astroid.Lambda)
+        """Prefer Operator Function to Lambda Functions"""
+
+        if isinstance(node.body, astroid.UnaryOp):
+            operator = OperatorUtils().get_unary_op(node.body.op)
+            varname = node.body.operand.name
+            lamfun = "lambda " + varname + ": " + node.body.op + " " + varname
+            opfun = "Operator." + operator
+            self.add_message('lambda-func', node=node, args={'op': opfun, 'lamfun': lamfun})
+        elif isinstance(node.body, astroid.BinOp):
+            self.__traverse_lambda(node.body)    
+
+    def __traverse_lambda(self, node):
+     #  if shopify_python.ast.count_tree_size(node) == 1:
+      #      print node.name
+        if shopify_python.ast.count_tree_size(node) == 3:
+            operator = OperatorUtils().get_bin_op(node.op)
```

Comment:
You could instead write this as:
```
BINARY_OPERATORS = {
...
}
...
operator = BINARY_OPERATORS.get(node.op)
if operator:
    ...
    
```

And save yourself the getter functions and make those dicts constants.